examples/react-native-playground/package.json

```diff
@@ -1,24 +1,53 @@
 {
   "name": "@suspensive/react-native-playground",
+  "main": "index.js",
   "version": "0.0.0",
   "private": true,
   "author": "Jonghyeon Ko <jonghyeon@toss.im>",
-  "main": "index.js",
   "scripts": {
-    "android": "expo start --android",
-    "eject": "expo eject",
-    "ios": "expo start --ios"
+    "start": "expo start",
+    "android": "DARK_MODE=media expo start --android",
+    "ios": "DARK_MODE=media expo start --ios",
+    "web": "DARK_MODE=media expo start --web",
+    "test": "jest --watchAll"
+  },
+  "jest": {
+    "preset": "jest-expo"
   },
   "dependencies": {
     "@suspensive/react-native": "workspace:*",
+    "@expo-google-fonts/noto-sans-kr": "^0.2.3",
+    "@expo/html-elements": "0.4.2",
+    "@expo/vector-icons": "^14.0.2",
+    "@legendapp/motion": "^2.4.0",
+    "@react-navigation/native": "^6.0.2",
+    "babel-plugin-module-resolver": "^5.0.2",
     "expo": "catalog:",
-    "expo-status-bar": "catalog:",
+    "expo-font": "~12.0.10",
+    "expo-linking": "~6.3.1",
+    "expo-router": "catalog:",
+    "expo-splash-screen": "~0.27.5",
+    "expo-status-bar": "~1.12.1",
+    "expo-system-ui": "~3.0.7",
+    "expo-web-browser": "~13.0.3",
+    "jscodeshift": "0.15.2",
+    "prettier": "^3.3.3",
     "react": "catalog:react18",
-    "react-native": "catalog:react18"
+    "react-native": "catalog:react18",
+    "react-dom": "catalog:react18",
+    "react-native-gesture-handler": "~2.16.1",
+    "react-native-reanimated": "~3.10.1",
+    "react-native-safe-area-context": "4.10.5",
+    "react-native-screens": "3.31.1",
+    "react-native-svg": "13.4.0",
+    "react-native-web": "~0.19.10"
   },
   "devDependencies": {
-    "@babel/core": "^7.26.0",
-    "@expo/webpack-config": "catalog:",
-    "@types/react": "catalog:react18"
-  }
+    "@babel/core": "^7.20.0",
+    "@types/react": "catalog:react18",
+    "jest": "catalog:",
+    "jest-expo": "catalog:",
+    "typescript": "~5.3.3"
+  },
+  "packageManager": "pnpm@9.12.0+sha512.4abf725084d7bcbafbd728bfc7bee61f2f791f977fd87542b3579dcb23504d170d46337945e4c66485cd12d588a0c0e570ed9c477e7ccdd8507cf05f3f92eaca"
```

Comment:
```suggestion
```

Reply:
That was my personal setting and I forgot to delete it.

I'll add a commit to delete that setting. Thank you!

Reply:
Thanks!

Reply:
I edited this part!